macros/sql/surrogate_key.sql

```diff
@@ -1,51 +1,19 @@
 {%- macro surrogate_key(field_list) -%}
-    {# needed for safe_add to allow for non-keyword arguments see SO post #}
-    {# https://stackoverflow.com/questions/13944751/args-kwargs-in-jinja2-macros #}
     {% set frustrating_jinja_feature = varargs %}
     {{ return(adapter.dispatch('surrogate_key', 'dbt_utils')(field_list, *varargs)) }}
 {% endmacro %}
 
 {%- macro default__surrogate_key(field_list) -%}
 
-{%- if varargs|length >= 1 or field_list is string %}
-
 {%- set error_message = '
-Warning: the `surrogate_key` macro now takes a single list argument instead of \
-multiple string arguments. Support for multiple string arguments will be \
-deprecated in a future release of dbt-utils. The {}.{} model triggered this warning. \
+Warning: the `dbt_utils.surrogate_key` macro has been fully deprecated in favor of \
+the new `generate_surrogate_key` macro. This new macro adjusts the default string value \
+passed to the coalesce function. To achieve parity with the original macro, \
+add a variable called `surrogate_key_treat_nulls_as_empty_strings` to your project with a \
+value of True. \
+The {}.{} model triggered this warning. \
```

Comment:
```suggestion
Warning: `dbt_utils.surrogate_key` has been replaced by \
`dbt_utils.generate_surrogate_key`. The new macro treats null values \
differently to empty strings. To restore the behaviour of the original \
macro, add a variable scoped to the dbt_utils package called \
`surrogate_key_treat_nulls_as_empty_strings` to your \
dbt_project.yml file with a value of True. \
The {}.{} model triggered this warning. \
```

Tweaked the wording. Eyeballed the alignment in the GitHub UI so it's probably not 80 chars exactly or whatever number you were going for.